### PR TITLE
Typehint discord/bot.py

### PR DESCRIPTION
**Pull request by Dorukyum**
## Checklist
- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by BobDotCom** [comment]
Upon approval from Swas or Middledot, conflicts can be resolved and immediately merged

File: discord/bot.py [hunk 1]
```diff
@@ -22,24 +22,25 @@
...
-from typing import Callable
+from typing import Callable, Dict, List, Optional
 
+from .app import (ApplicationCommand, MessageCommand, SlashCommand,
+                  SubCommandGroup, UserCommand)
 from .client import Client
+from .interactions import Interaction
 from .shard import AutoShardedClient
 from .utils import get
-from .app import SlashCommand, SubCommandGroup, MessageCommand, UserCommand, ApplicationCommand
```
Comment (Middledot) [inline discord/bot.py:34]: Why's this removed?

Comment (Dorukyum) [reply discord/bot.py:34]: It has been moved to line 29.

**Comment by CodeWithSwastik** [comment]
I'll do these myself. The code seems to be an older version.

**Pull request closed without merging**